nautobot/docs/additional-features/jobs.md

```diff
@@ -553,57 +553,85 @@ Provision of user input (`data` values) via the CLI is not supported at this tim
+### Nautobot 1.3 and later
 
-1. Inherit from `django.test.TransactionTestCase` instead of `django.test.TestCase`. (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes.)
-2. Set your test class's `databases` attribute to `("default", "job_logs")`. (`job_logs` is a proxy connection to the same (`default`) database that's used exclusively for Job logging.)
+The simplest way to test the entire execution of Jobs from 1.3 on is via calling the `nautobot.utilities.testing.run_job_for_testing()` method, which is a helper wrapper around the `run_job` function used to execute a Job via Nautobot's Celery worker process.
 
-!!! tip
-    Additionally, *if your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x*, you should add the following block to your test module:
-
-        from django.conf import settings
+Because of the way `run_job_for_testing` and more specifically `run_job()` works, which is somewhat complex behind the scenes, you need to inherit from `nautobot.utilities.testing.TransactionTestCase` instead of `django.test.TestCase` (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes - `TransactionTestCase` from Nautobot is a small wrapper around Django's `TransactionTestCase`).
 
-        if "job_logs" in settings.DATABASES:
-            settings.DATABASES["job_logs"] = settings.DATABASES["job_logs"].copy()
-            settings.DATABASES["job_logs"]["TEST"] = {"MIRROR": "default"}
+### Nautobot 1.2
 
-    This code isn't *needed* in Nautobot 1.3.x and later, as the `"job_logs"` database will already be set up correctly for testability, but it also won't hurt anything to include.
-
-A simple example of a Job test case that's implemented to be backwards-compatible with Nautobot 1.2.x might look like the following:
+If your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x, you need to handle a couple more things manually:
 
+Set up the `"job_logs"` database correctly for testing:
 ```python
-import uuid
-
 from django.conf import settings
-from django.contrib.contenttypes.models import ContentType
-from django.test import TransactionTestCase
-
-from nautobot.extras.jobs import run_job
-from nautobot.extras.models import Job, JobResult, JobLogEntry
-
 
 if "job_logs" in settings.DATABASES:
     settings.DATABASES["job_logs"] = settings.DATABASES["job_logs"].copy()
     settings.DATABASES["job_logs"]["TEST"] = {"MIRROR": "default"}
+  ```
+Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test:
+```python
```

Comment:
````suggestion
```

Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the Celery worker would run the test:

```python
````

More generally, we have `invoke markdownlint` now available in `develop` - please run it and fix any other issues it reports.